bin/scripts/org_migrator/org_migrator.py

```diff
@@ -455,7 +456,8 @@ def do_export(self):
 
         content_uuid_query1 = 'SELECT oc.content_uuid FROM cp2_owner_content oc WHERE oc.owner_id = %s' + \
             'UNION ' + \
-            'SELECT ec.content_uuid FROM cp2_environment_content ec JOIN cp_environment e on e.id = ec.environment_id WHERE e.owner_id = %s'
+            'SELECT cont.uuid FROM cp_environment e JOIN cp_environment_content ec ON e.id = ec.environment_id  ' + \
```

Comment:
We still don't want to do this query at all anymore -- It's no longer required for pulling the data, and this query doesn't guarantee it'll pull the same thing on every export.

Same with the product version